build.gradle.kts

```diff
@@ -68,6 +68,7 @@ dependencies {
 
     testImplementation(kotlin("test"))
     testImplementation("io.ktor:ktor-server-test-host:$ktorVersion")
+    testImplementation("com.squareup.okhttp3:mockwebserver:3.13.1")
```

Comment:
Added https://github.com/square/okhttp/tree/master/mockwebserver dependency. This is similar to Wiremock, but Wiremock currently do not work against ktor jetty dependency (Ktor use jetty 9.4.x and wiremock 9.2.x - they are not combatible)